### ISSUE TITLE
Unable to set debug breakpoint in methods with sequence points

### ISSUE THREAD

**Issue by psaxton** [vescon/MethodBoundaryAspect.Fody#101]
## Problem
Setting a break point in Visual Studio or VS Code within a decorated method with sequence points results in the breakpoint being ignored.

## Expected Result
The debugger should be able to find the breakpoint and break.

## Minimum reproducible code
```pwsh
mkdir \temp\mba-test
cd \temp\mba-test
dotnet new console
dotnet add package Fody
dotnet add package MethodBoundaryAspect.Fody
code .
```

Program.cs
```csharp
using MethodBoundaryAspect.Fody.Attributes;

public static class Program
{
    private static void Main()
        => new Interactor().DoSomething();
}

public class Interactor
{
    [DummyAspect]
    public void DoSomething()
    {
        if (true)
        {
            Console.WriteLine("Hello world");
        }
    }
}

public class DummyAspect : OnMethodBoundaryAspect
{
    public override void OnEntry(MethodExecutionArgs arg)
        => base.OnEntry(arg);
}

```

From Visual Studio or VS Code, set a breakpoint within Interactor.DoSomething() (Lines 13-18). Run the program from IDE. The breakpoint is never hit.

This appears to be a regression with v2.0.146. Setting the MethodBoundaryAspect.Fody version to 2.0.145 produces the expected result.

**Comment by Ralf1108** [comment]
This really seems to be an regression. So with your test case this should be reproducible

**Comment by Ralf1108** [comment]
commit was [9d451fbff2290110dc53d0a82caacb744fe782f4](commit:9d451fbff2290110dc53d0a82caacb744fe782f4)

**Comment by psaxton** [comment]
Confirmed, tested with that single commit reverted [issue101/rollback-9d451fb](https://github.com/psaxton/MethodBoundaryAspect.Fody/tree/issue101/rollback-9d451fb). Should I create a pull request?

**Comment by Ralf1108** [comment]
fixed in v2.0.148
